prow/cluster/build/400-boskos-janitor.yaml

```diff
@@ -21,7 +21,7 @@ metadata:
     app: boskos-janitor
   namespace: test-pods
 spec:
-  replicas: 6  # Number of distributed janitor instances; k8s has ~1/16 as many projects
+  replicas: 4  # Number of distributed janitor instances; k8s has ~1/16 as many projects
```

Comment:
Why reducing this number?

Reply:
I'm thinking fewer replicas but larger ones. I can revert it if it won't work.